src/launchpads/MK2/LaunchpadMK2.ts

```diff
@@ -1,4 +1,4 @@
-import { CONTROL_NOTE, NORMAL_NOTE } from '../../utils.js';
+import { CONTROL_NOTE, NORMAL_NOTE } from '../../internal/utils.js';
```

Comment:
I think you forgot to add the file to git after you moved it ;)

Reply:
Doh